OakCityV2/Job.h

```diff
@@ -0,0 +1,21 @@
+//
+//  DataObject.h
+//  OakCityV2
+//
+//  Created by Josh Green on 6/29/15.
+//  Copyright (c) 2015 Josh Green. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Job : NSObject
+
+@property (nonatomic, readwrite) NSString *title;
+@property (nonatomic, readwrite) NSString *company;
+@property (nonatomic, readwrite) NSString *desc;
+@property (nonatomic, readwrite) NSString *url;
+@property (nonatomic, readwrite) NSString *logo;
+
```

Comment:
We want to represent that the attributes of a Job are immutable after the Job is created, so these should be readonly attributes.

If you want to be extra precise, the url properties (url and logo) could be NSURL attributes instead of strings.